### PR TITLE
OCPBUGS-30254: Updated the vSphere version for 4.14 onwards

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.14 +

Issue:
[OCPBUGS-30254](https://issues.redhat.com/browse/OCPBUGS-30254)

Link to docs preview:
[VMware vSphere infrastructure requirements](https://72608--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/ipi-vsphere-installation-reqs#installation-vsphere-infrastructure_ipi-vsphere-installation-reqs)

- [x] SME has approved this change.
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-infrastructure.adoc [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
+You must install an {product-title} cluster on one of the following versions of a VMware vSphere instance that meets the requirements for the components that you use:
 
-[NOTE]
-====
-{product-title} version {product-version} supports VMware vSphere version 8.0.
-====
+* Version 7.0 Update 3 or later
+* Version 8.0 Update 2 or later
+
+Both of these releases support Container Storage Interface (CSI) migration, which is enabled by default on {product-title} {product-version}.
```
Comment (dfitzmau) [inline modules/installation-vsphere-infrastructure.adoc:14]: @jcpowermac . OK to mention  CSI migration here?

Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc:14]: Just wondering if this [doc note](https://docs.openshift.com/container-platform/4.15/storage/persistent_storage/persistent-storage-vsphere.html#:~:text=For%20new%20installations%2C%20OpenShift%20Container%20Platform%204.13%20and%20later%20provides%20automatic%20migration%20for%20the) is no longer valid?

Comment (jcpowermac) [reply modules/installation-vsphere-infrastructure.adoc:14]: @jsafrane can you take a look?

**Comment by jcpowermac** [comment]
@racedo can you take a look as well. The concern was the disparity between CSI migration requirements and installation.

**Comment by duanwei33** [comment]
LGTM
(From storage QE perspective)

**Review by Srivaralakshmi — COMMENTED** [review]
@dfitzmau Nice work! Out of scope, but you must update the content type for [Module file metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata).

I will not hold the merge. However, I suggest raising a JIRA issue to address this comment. Apply this comment for all such instances in our component. Thank you!

**Pull request merged**